engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetViewportToSizeOf.java

```diff
@@ -86,7 +80,7 @@ public void update() {
 
     @Override
     public String toString() { // TODO: used for logging purposes at the moment, investigate different methods
-        return String.format("%30s: %s (%sx%s)", this.getClass().getSimpleName(), fboName, fboWidth, fboHeight);
+        return String.format("%30s: %s (%sx%s)", this.getClass().getSimpleName(), fbo.fboId, fboWidth, fboHeight);
     }
 
     public static void disposeDefaultInstance() {
```

Comment:
Yes, we can write it so. As you said, there *might* be a tiny loss in performance, and I'm not sure if JIT will optimize it.. but I don't think the performance loss will be anywhere near noticeable, even if we time it to microseconds.

I'll do this change in a separate commit, and we can decide if we want to keep it :D